docs/pages/push-notifications/overview.mdx

```diff
@@ -1,147 +1,42 @@
 ---
-title: Push Notifications Overview
+title: Push notifications overview
 sidebar_title: Overview
 ---
 
-Push Notifications are an important feature, no matter what kind of app you're building. Not only is it nice to let users know about something that may interest them, be it a new album being released, a sale or other limited-time-only deal, or that one of their friends sent them a message, but push notifications are proven to help boost user interaction and create a better overall user experience.
+import { BoxLink } from '~/ui/components/BoxLink';
 
-Whether you just want to be able to let users know when a relevant event happens, or you're trying to optimize customer engagement and retention, Expo makes implementing push notifications almost too easy. All the hassle with native device information and communicating with APNs (Apple Push Notification service) or FCM (Firebase Cloud Messaging) is taken care of behind the scenes, so that you can treat iOS and Android notifications the same, saving you time on the front-end, and back-end!
+Push notifications are an important feature, no matter what kind of app you're building. They are a proven way to help boost user interaction and provide a better user experience by letting app users know about relevant events.
```

Comment:
```suggestion
```
I would argue that we don't need an overview/stated reason for the existence of notifications. I think people know what these are and we can dive right in wit the next paragraph.